backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -594,9 +614,9 @@ void addTestResult_correction() {
         LocalDate.of(1865, 12, 25),
         LocalDate.ofInstant(
             correctionTestEvent.getDateTested().toInstant(), ZoneId.systemDefault()));
-    assertEquals(2, _resultRepository.findAllByTestOrder(response.getTestOrder()).size());
-    assertEquals(1, _resultRepository.findAllByTestEvent(correctionTestEvent).size());
-    assertEquals(1, _resultRepository.findAllByTestEvent(originalTestEvent).size());
+    assertEquals(6, _resultRepository.findAllByTestOrder(response.getTestOrder()).size());
```

Comment:
Not sure if we actually want the test result size assertions to get multiplied by 3 here. Lmk If I should be constructing the test events in another way!